numbert/reranker.py

```diff
@@ -8,6 +8,12 @@
 from .utils.utils_numbert import (processors, output_modes,
                                     convert_examples_to_features)
 
+try:
+    from tokenizers import BertWordPieceTokenizer
```

Comment:
Maybe print a warning in case `tokenizers` is not available? Also, maybe just add `tokenizers` to the requirements and let python raise an exception in case it is not installed?

Reply:
I basically wanted it to run without the need for tokenizers (mostly because few places like ComputeCanada seem to have an issue getting tokenizers running so it should default to the regular slower tokenizer in that case. But since tokenizers is already in the requirements, I should probably just remove this.

Reply:
Fixed in latest commit.